visualizations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -84,21 +84,19 @@ def make_observation(obs, index):
            index
         ],
         'showlegend': False,
-        'legendgroup': obs['name'],
```

Comment:
Why remove this? This causes the marker to be hidden when you unselect the corresponding line legend. Seems like the appropriate solution to me.

Reply:
The idea is to have observations be filtered instead and not be bound to a 'name'. I.e. If you choose the `'temperature'` category you want all the temperature predictions (fans) and the temperature observations, but the observations arent bound to any prediction(fan/legend). Instead they are all black and represent all temperature observations.